install/kubernetes/helm/istio/charts/security/templates/enable-mesh-permissive.yaml

```diff
@@ -0,0 +1,17 @@
+{{ define "permissive-default.yaml.tpl" }}
+# Authentication policy to enable permissive mode for all services (that have sidecar) in the mesh.
+# In this case, all services can take both http and mTLS traffic.
+apiVersion: "authentication.istio.io/v1alpha1"
+kind: "MeshPolicy"
+metadata:
+  name: "default"
+  labels:
+    app: istio-security
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  peers:
+  - mtls:
+      mode: PERMISSIVE
+{{- end }}
```

Comment:
this file is the same as install/kubernetes/helm/istio-remote/charts/security/templates/enable-mesh-permissive.yaml. A symlink pointing to the same file should have worked. But it might be done in a different PR.